src/components/homepages/ProductNewsCard.tsx

```diff
@@ -0,0 +1,162 @@
+import Markdown from 'components/Markdown';
+import * as React from 'react';
+import styled from 'styled-components'
+
+const CardBaseDiv = styled.div`
+    box-sizing: border-box;
+    margin: 0;
+
+    padding: 40px 30px 30px;
+    background: #FFFFFF;
+
+    border: 1px solid #E2E2E2;
+
+    box-shadow: 3px 3px 7px rgba(0, 0, 0, 0.08);
+    border-radius: 5px;
+    :not(:last-child) {
+        margin-bottom: 36px;
+    }
+    font-family: 'Helvetica';
+    font-style: normal;
+    display: flex;
+    flex-direction: column;
+`;
+const CardTitle = styled.h2`
+    font-weight: 700 !important;
+    font-size: 24px !important;
+    line-height: 28px;
+    color: #003B45;
+    margin: 0 0 3px 0;
+`;
+
+const CardDate = styled.p`
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 150%;
+    text-transform: uppercase;
+    color: #858585;
+    margin: 0 0 10px 0;
+`;
+
+const CardTags = styled.span`
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 5px 15px;
+
+    width: fit-content;
+
+    background: #EBF9EA;
+    border-radius: 12px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #00A176;
+`
+
+const CardMarkdown = styled.div`
+    font-family: 'Helvetica';
+    font-style: normal;
+    font-size: 14px;
+    line-height: 150%;
+    color: #575757;
+    & h3 {
+        font-weight: 700;
+        color: #003B45;
+        font-size: 20px;
+    };
+    & h4 {
+        font-weight: 700;
+        color: #54909B;
+        font-size: 16px;
+    }
+    margin-bottom: 20px;
+    padding-top: 10px;
+    border-top: 1px solid #003B45;
+`;
+
+const CardButton = styled.a`
+    width: fit-content;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+    padding: 8px 10px;
+    gap: 10px;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    color: #FFFFFF !important;
+    background: #00A176;
+    border-radius: 5px;
+    margin-bottom: 0;
```

Comment:
```suggestion
    margin-bottom: 0;
    margin-top: 20px;
```

Have the margin between text and button only when the button is rendered